fetch.md

```diff
@@ -238,8 +238,8 @@ proxy server or virtual private network, if available)
 [1]: https://html.spec.whatwg.org/multipage/webappapis.html#concept-environment-top-level-origin
 [2]: https://fetch.spec.whatwg.org/#network-partition-key
 [3]: https://fetch.spec.whatwg.org/#concept-request-cache-mode
-[4]: https://htmlpreview.github.io/?https://raw.githubusercontent.com/jeremyroman/alternate-loading-modes/spec/index.html#prerendering-browsing-context
+[4]: https://htmlpreview.github.io/?https://raw.githubusercontent.com/WICG/nav-speculation/spec/index.html#prerendering-browsing-context
```

Comment:
This link seems subpar and maybe broken; should we fix it while here?

Reply:
```suggestion
[4]: https://wicg.github.io/nav-speculation/prerendering.html#prerendering-browsing-context
```